CALENDAR_VERSION

```diff
@@ -0,0 +1 @@
+2021.09
```

Comment:
Not something we have to change in this PR, but we're starting to accumulate a large number of these files at the repo root. I wonder if we should just have one that contains calendar version, flower, build type, etc.

Reply:
I think that would add complexity to the parsing system. What about moving them to their own folder, like `version` or `release` or something?